packages/docusaurus-theme-classic/src/theme/DocSidebar/Desktop/index.tsx

```diff
@@ -24,16 +24,19 @@ function DocSidebarDesktop({path, sidebar, onCollapse, isHidden}: Props) {
   } = useThemeConfig();
 
   return (
-    <div
+    <nav
       className={clsx(
         styles.sidebar,
         hideOnScroll && styles.sidebarWithHideableNavbar,
         isHidden && styles.sidebarHidden,
-      )}>
-      {hideOnScroll && <Logo tabIndex={-1} className={styles.sidebarLogo} />}
+      )}
+      aria-label="docs sidebar">
```

Comment:
All our Aria labels should use our translation framework so that they can be localized